chapter06/jxp/ch6ex1.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use feature "say";
+
+my %names = qw( fred flintstone barney rubble wilma flintstone );
```

Comment:
I like that using this format makes it easier to read that persons name (rather than reading 'fred => flinstone'), but having it all on one line makes it a little bit tough to glance for the nth entry